pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java

```diff
@@ -92,6 +93,28 @@
         }
     }
 
+    @GET
+    @Path("/leaderBroker")
+    @ApiOperation(
+            value = "Get the service url of the leader broker in the cluster.",
+            response = String.class)
+    @ApiResponses(
+            value = {
+                    @ApiResponse(code = 401, message = "Authentication required"),
+                    @ApiResponse(code = 403, message = "This operation requires super-user access") })
+    public String getLeaderBroker() throws Exception {
```

Comment:
What about returning a structure instead of a single String?
This way in the future we will be about to return more information .
If we return only one String we cannot extend this API in the future

Reply:
Thanks for your tip,. 
What about returning LeaderBroker directly?

Reply:
@eolivelli Good suggestion. 

Yes, I think you can use the class LeaderBroker directly.

Reply:
+1 to return LeaderBroker